src/test/java/edu/harvard/iq/dataverse/api/HarvestingClientsIT.java

```diff
@@ -272,6 +270,12 @@ private void harvestingClientRun(boolean allowHarvestingMissingCVV)  throws Inte
         } while (i<maxWait); 
         
         System.out.println("Waited " + i + " seconds for the harvest to complete.");
+
+        Response searchHarvestedDatasets = UtilIT.search("metadataSource:" + nickName, normalUserAPIKey);
```

Comment:
Let's keep an eye on it - will not surprise me if it needs an extra second here, occasionally, for the indexing of all the harvested datasets to complete.

Reply:
@jp-tosca @stevenwinship 
Hah! This appears to be exactly what's happening now, since I merged this PR yesterday. 
The new, weird failures - ```data.total_count doesn't match. Expected: <8>  Actual: <5>```

Reply:
JP, this was a bad call on my part - I should've asked you to run Jenkins 3 times in a row on the PR before merging it.